svg/_filters.py

```diff
@@ -251,6 +251,17 @@ class FeMorphology(Element, m.FilterPrimitive):
     height: Length | Number | None = None
 
 
+@dataclass
+class FeDropShadow(Element, m.FilterPrimitive):
+    element_name = "feDropShadow"
+    dx: Optional[Any] = None
+    dy: Optional[Any] = None
+    flood_opacity: Optional[Number] = None
+    flood_color: Optional[str] = None
```

Comment:
I think it doesn't have `flood_opacity` and `flood_color`. Where did you get it? 🤔 

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feDropShadow

Reply:
Actually right from there on that very MDN page, the fourth entry under `Global Attributes` are `Presentation Attributes` with the only line being "`Most notably: flood-color, flood-opacity`". 

I accounted for all the other Global Attributes in the base Element dataclass but these 2 weren't in Element (as expected) nor in the FilterPrimitive mixin so I added them here (and actually *do* use them on my DropShadow effect!).

Reply:
Ah, I see! Sorry, I missed it somehow, I looked only at base attributes.